policyengine_us/parameters/gov/irs/gross_income/sources.yaml

```diff
@@ -21,8 +21,8 @@ values:
     - illicit_income
     - taxable_retirement_distributions
     - miscellaneous_income
-    # Listed separately on 1040 line 8g.
     - ak_permanent_fund_dividend
+    - tip_income
```

Comment:
This double-counts since it's part of employment_income.

Considering also Trump's overtime exemption proposal, I'd suggest one of two options:
1. Break out all income sources in the main codebase: non-OT wages & salaries, OT wages & salaries, tip income
2. Have OT and tip income live only in the reform code, and then calculate the remainder by subtracting them from employment_income